Source/Core/DolphinWX/NetPlay/NetPlayLauncher.cpp

```diff
@@ -92,32 +92,22 @@ bool NetPlayLauncher::Join(const NetPlayJoinConfig& config)
   }
 }
 
-const std::string NetPlayLaunchConfig::DEFAULT_TRAVERSAL_HOST = "stun.dolphin-emu.org";
-
-std::string
-NetPlayLaunchConfig::GetTraversalHostFromIniConfig(const IniFile::Section& netplay_section)
+std::string NetPlayLaunchConfig::GetTraversalHost()
 {
-  std::string host;
-
-  netplay_section.Get("TraversalServer", &host, DEFAULT_TRAVERSAL_HOST);
-  host = StripSpaces(host);
+  std::string host = StripSpaces(Config::Get(Config::NETPLAY_TRAVERSAL_SERVER));
 
   if (host.empty())
-    return DEFAULT_TRAVERSAL_HOST;
+    return Config::NETPLAY_DEFAULT_TRAVERSAL_HOST;
```

Comment:
Can this be the default value for Config::NETPLAY_TRAVERSAL_SERVER? (I think the StripSpaces is also unnecessary as IniFile will do that.)